packages/launchpad/src/checklist-item/style.scss

```diff
@@ -180,8 +201,8 @@
 }
 
 // pending tasks and completed enabled
-.checklist-item__task.completed.enabled:hover,
-.checklist-item__task.pending:hover,
+.checklist-item__task.completed.enabled > button:hover,
```

Comment:
We only want the `hover` on the button. Previously we didn't have anything else inside the `li`, but now we have the expandable content. And we don't want to apply the hover effect when hovering the expandable content.